libraries/eosiolib/contracts/eosio/privileged.hpp

```diff
@@ -185,6 +212,23 @@ namespace eosio {
     */
    void get_blockchain_parameters(eosio::blockchain_parameters& params);
 
+   /**
+    *  Set the blockchain parameters flexibly by id data pair vector
+    *
+    *  @ingroup privileged
+    *  @param params - New blockchain parameters to set, only id and data in the params will be set.
+    */
+   void set_parameters(const std::vector<std::pair<uint32_t, std::variant<uint16_t, uint32_t, uint64_t>>>& params);
```

Comment:
How about create an alias for `std::vector<std::pair<uint32_t, std::variant<uint16_t, uint32_t, uint64_t>>>` for here and other related code by `using`?

Reply:
Sure.